lib/templateCompilerModules/utils.ts

```diff
@@ -6,6 +6,8 @@ export interface Attr {
 export interface ASTNode {
   tag: string
   attrs: Attr[]
+  staticStyle?: string
```

Comment:
I am not convinced in changing the interface of ASTNode, if not necessary to surface staticStyle from attrs. Also now codegen has to modified to treat styles differently.

Reply:
>I am not convinced in changing the interface of ASTNode, it's not necessary to surface staticStyle from attrs.

since `attrs` doesn't contain `style`

>Also now codegen has to modified to treat styles differently.

Agree